[CUBRIDMAN-25] manual for change column comments without writing column definitions (#226)
CUBRID/cubrid-manual: en/sql/schema/view_stmt.rst

COMMENT Clause
--------------

> View and column comments can be changed by using the **COMMENT** clause of the **ALTER VIEW** syntax.

Reviewer: how about?
```suggestion
You can change a view's comment and column comments with **COMMENT** clause of **ALTER VIEW** syntax.
```

> You can change a view's comment and column comments with **COMMENT** clause of **ALTER VIEW** syntax.

Reviewer 2: You can change a view's comment and column comments with
-->
You can change a view's comment,  columns' comment, or attributes' comment with ...
apply also to Korean version.

Author: Thank you. Fixed in '5228902' commit.

::

    ALTER [VIEW | VCLASS] view_name
    COMMENT [=] 'view_comment';
    COMMENT ON {COLUMN | CLASS ATTRIBUTE} <column_comment_definition> [, <column_comment_definition>] ;

        <column_comment_definition> ::= column_name [=] 'column_comment_string'

*   *view_name*: specifies the name of a view to be modified.
*   *column_name*: specifies the name of a column to be modified.

> *   *view_comment_string*: specifies a view's comment.

Reviewer: 'view_comment_string' is not exists

Author: Thank you. The error was corrected by changing 'view_comment' to 'view_comment_string' in BNF.

> *   *column_comment_string*: specifies a column's comment.

Reviewer 3: specifies --> Specifies
(Since the other part of the manual use capital letter after ':', I think is better in there, we also capitalise the word 'specifies')

Author: Thank you. Fixed in '1989cf0' commit.

The following example shows how to change a view's comments.

.. code-block:: sql

    ALTER VIEW v1 COMMENT = 'changed view v1 comment';

You can change the column comment by specifying one or more columns after the ON COLUMN keyword.
The following example shows how to change a column's comments.

.. code-block:: sql

    ALTER VIEW v1 COMMENT ON COLUMN c1 = 'changed view column c1 comment';
    ALTER VIEW v1 COMMENT ON COLUMN c2 = 'changed view column c2 comment', c3 = 'changed view column c3 comment';

> The below is a syntax to show a column's comment.

Reviewer 3: The below is a syntax to show a column's comment. -->  Below is a syntax to show a column's comment.  (Delete 'The')

Author: Thank you. Fixed in '1989cf0' commit.
